src/components/new-request/search.vue

```diff
@@ -2,7 +2,7 @@
   <v-container fluid id="search-container" class="copy-normal pt-10 px-10 pb-12">
     <v-row>
       <v-col cols="12" class="font-weight-bold h6">
-        Get started by selecting an action: [{{ getRequestActionCd }} / {{ getLocation }} / {{ entity_type_cd }}]
```

Comment:
Move to Landing.vue, ie:

![image](https://github.com/bcgov/namerequest/assets/10002889/aeaad9b4-fc38-4946-97aa-3c1b56a632c8)